pkg/channel/distributed/common/kafka/util/util.go

```diff
@@ -20,17 +20,26 @@ import (
 	"fmt"
 	"strings"
 
+	"k8s.io/apimachinery/pkg/types"
+
 	"knative.dev/eventing-kafka/pkg/channel/distributed/common/kafka/constants"
 )
 
+const GroupIdPrefix = "kafka"
```

Comment:
Seems like the prefix is "kafka." - why not include the dot here instead of having to add it both in GroupId() and Uid() and the tests ?

Reply:
I originally had it that way but felt the "." was a "separator" instead of the prefix - I think it's ok as is.